scripts/build-linux.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+if [ "$(whoami)" = "root" ]
+then
+    echo "Running the script as root is not permitted"
+    exit 1
+fi
+
+CALLED=$_
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && SOURCED=1 || SOURCED=0
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+
+if [ $SOURCED = 1 ]; then
+        echo "You must run this script, rather then try to source it."
+        echo "$SCRIPT_SRC"
+        return
+fi
+
+if [ -z "$HDMI2USB_ENV" ]; then
+        echo "You appear to not be inside the HDMI2USB environment."
+	echo "Please enter environment with:"
+	echo "  source scripts/enter-env.sh"
+        exit 1
+fi
+
+# Imports TARGET, PLATFORM, CPU and TARGET_BUILD_DIR from Makefile
+export FIRMWARE=linux
+eval $(make env)
+make info
+
+set -x
+set -e
+
+if [ "$CPU" != or1k ]; then
+	echo "Linux is only supported on or1k at the moment."
+	exit 1
+fi
+
+(
+	cd third_party/litex
+	git checkout or1k-linux
```

Comment:
Why use `or1k-linux` here, but `litex-minimal` below?

Reply:
This is checking out in `third_party/litex` directory - below is checking out the Linux repo into `third_party/linux`.

Reply:
Ah, litex/linux look almost the same to me.